plugins/resin/pom.xml

```diff
@@ -24,13 +24,13 @@
 			<artifactId>javax.servlet-api</artifactId>
 			<scope>compile</scope>
 		</dependency>
-
 		<dependency>
-			<groupId>com.caucho</groupId>
-			<artifactId>resin</artifactId>
-			<version>4.0.35</version>
-			<scope>provided</scope>
+		    <groupId>com.caucho</groupId>
+		    <artifactId>resin</artifactId>
+		    <version>3.1.15</version>
```

Comment:
Let's use the version available from maven central - `3.0.9`.
(Pinpoint users shouldn't have to install resin into their local repository)